workbench-core/example/infrastructure/integration-tests/support/resources/dynamicAuthorization/identityPermissions.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  AuthenticatedUser,
+  CreateIdentityPermissionsRequest,
+  Identity,
+  IdentityPermission as Permission
+} from '@aws/workbench-core-authorization';
+import { AxiosResponse } from 'axios';
+import ClientSession from '../../clientSession';
+import RandomTextGenerator from '../../utils/randomTextGenerator';
+import CollectionResource from '../base/collectionResource';
+import IdentityPermission from './identityPermission';
+
+function createPermissionId(identityPermission: Permission): string {
+  const { subjectId, subjectType, identityId, identityType, effect, action } = identityPermission;
+  return `${subjectType}|${subjectId}|${action}|${effect}|${identityType}|${identityId}`;
+}
+
+export default class IdentityPermissions extends CollectionResource {
+  public constructor(clientSession: ClientSession) {
+    super(clientSession, 'identityPermissions', 'identityPermission');
+    this._api = 'authorization';
+  }
+
+  public identityPermission(identityPermission: Permission, id: string): IdentityPermission {
+    return new IdentityPermission(identityPermission, this._clientSession, this._parentApi, id);
```

Comment:
I think `this._parentApi` should be `this._api` but I'm not 100% on that

Reply:
there is no parent api to the the each identity permission, I will make it an empty string to be precise